recipes/edffile/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "5.0.0" %}
+
+package:
+    name: edffile
+    version: {{ version }}
+
+source:
+    path: src
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - numpy  # does not compile against numpy's ABI. no 'x.x' needed
+
+    run:
+        - python
+        - numpy  # does not compile against numpy's ABI. no 'x.x' needed
+
+test:
+    # Python imports
+    imports:
+        - EdfFile
+
+about:
+    home: https://github.com/vasole/pymca/blob/master/PyMca5/PyMcaIO/EdfFile.py
```

Comment:
@ericdill is this the same case as `spefile` and `pyespec`?

That is a new precedent for packaging. It would be better if you add some info (maybe the description field?) explaining the "subset fork."